store/migrations/migration1549496047/migrate.go

```diff
@@ -0,0 +1,16 @@
+package migration1549496047
+
+import (
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/smartcontractkit/chainlink/store/orm"
+)
+
+type Migration struct{}
+
+func (m Migration) Timestamp() string {
+	return "1549496047"
+}
+
+func (m Migration) Migrate(orm *orm.ORM) error {
```

Comment:
Thought that we would not need a migration for this because it hasn't been run anywhere yet, so the new table could be tacked onto migration0. I'm not opposed to adding a migration, but I'm wondering how much this locks us into a pattern moving forward. How hard will it be to move this to a different migration system later?

Reply:
I personally think it will be trivial. It's a separate migration to keep things smooth and without friction for us developers. I don't think it locks us into a migration strategy.

Reply:
Looked like it, but you understand it better than me at this point. Sounds good.